src/likesAPI.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-console */
+import http from './http.js';
+
+// links
+const appKey = 'iTwOeYcQW4Syd1iyW4ip';
+const baseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi';
+const likesLink = `/apps/${appKey}/likes/`;
+// const commentLink = `/apps/${appKey}/comments`;
+// const reservationsLink = `/apps/${appKey}/reservations`;
+
+// Get Likes Count
+const getLikes = async () => {
+  const result = await http.get(`${baseUrl}${likesLink}`);
+  // const likes = await result.json();
+  console.log(result);
+  return result;
+};
```

Comment:
when I click on some of the like, I get `NaN`, this incident occur just to some few but every other ones work just fine, please kindly fix this

Here's a screenshot of it

<img width="1244" alt="Screenshot 2021-08-14 at 21 08 22" src="https://user-images.githubusercontent.com/34798898/129459007-dbcd747c-acf8-4a21-9862-c4f437ee85a0.png">

Reply:
Hello and thanks for the review.  This happens when you have a parameter searching for a movie. For some reason when I regenerate the movies from the search index likes count breaks. But in the default index, it works as you will see in the video i will implement on the PR. I will delete the search box the avoid any confusion about the real index.

Reply:
Hi @pountzas,

I can't see the search box there anymore but when I click on like for one item and click the next, I get the `NaN`, not sure this is related to search, I get your logic there is not well implemented, please kindly fix this

Reply:
- [ ] Like count seems like getting updated on the page now; but The like change doesn't get updated when I refresh the page. 

I checked the post request and looks like it sends `item_id	"movie-box"` to the server regardless of the movie that I clicked.